jwst/tests_nightly/general/pipelines/test_image2pipeline_1.py

```diff
@@ -20,7 +20,7 @@ def test_image2pipeline1(_bigdata):
                         )
 
     h = pf.open('jw00001001001_01101_00001_mirimage_cal.fits')
-    href = pf.open(_bigdata+'/miri/test_image2pipeline/jw00001001001_01101_00001_MIRIMAGE_uncal_MiriSloperPipeline_Image2Pipeline.fits')
+    href = pf.open(_bigdata+'/miri/test_image2pipeline/jw00001001001_01101_00001_mirimage_cal_ref.fits')
```

Comment:
Yeah, I recently (finally) cleaned up the name of that comparison file over on the pandokia side.